src/client/pages/OfferNew/Perils/PerilCollection/PerilItem.tsx

```diff
@@ -113,7 +113,7 @@ export const PerilItem: React.FC<PerilItemProps> = ({
   return (
     <OuterContainer>
       <Container onClick={onClick}>
-        {iconUrl && <PerilIcon src={iconUrl} />}
+        {iconUrl && <PerilIcon src={iconUrl} alt="" width={24} height={24} />}
```

Comment:
Since these icons are set to be `3 rem` in `width` and `height` on desktop with CSS won't this just be confusing to the poor browser and everyone else? Also, I don't really understand why we use `img` elements for the SVG's but that's definitely not something you decided 🙃

Reply:
Yeah I think in this case it will not make a difference. Perhaps the only reason is to be consistent and always add height/width to img-elements like Google recommends.

Agree that we can rework how we deal with SVG icons!

Reply:
Let's keep it like this "for now" then 😄 If there won't be an issue that the browser calculates the image size incorrectly here